test/4.public.UrlChecker.js

```diff
@@ -139,9 +139,83 @@ describe("PUBLIC -- UrlChecker", function()
 			}).enqueue( conn.absoluteUrl );
 		});
 	});
-	
-	
-	
+
+
+
+	describe("caching", function()
+	{
+		var options;
+		beforeEach(function()
+		{
+			options = utils.options();
```

Comment:
And, come to think of it, this probably doesn't need to be in a `beforeEach()`. It can probably go [redundantly] in each test -- it avoids the need to scroll around the document.